src/types/backEnd/PollutedLocationGetResponse.ts

```diff
@@ -1,13 +1,12 @@
 import PollutedLocation, { severityLevels } from "../PollutedLocation";
 
-/**
- * Data Transfer Object for @type {PollutedLocation}.
- */
-type PollutedLocationDTO = Partial<{
+type PollutedLocationGetRequest = Partial<{
```

Comment:
the fileName is GetResponse, but you define GetRequest here

Reply:
I believe we should match the naming from be. We're sending a request with some params and receiving a response. This current naming would contradict that.

Reply:
haha yes sorry, I mixed it up, will change to `PollutedLocationResponse` same as in BE